notebook_data_redirector/sync.py

```diff
@@ -11,13 +11,21 @@ def lambda_handler(event, context):
     ddb_table = common.get_ddb_table()
     box_client, _ = common.get_box_client()
     root_folder = box_client.folder(common.BOX_FOLDER_ID)
+    root_shared = common.is_box_file_public(root_folder)
```

Comment:
It may be worth renaming this function to `is_box_object_public` or something, just so it's clear that it can handle both files and folders.

Reply:
done